includes/html/functions.inc.php

```diff
@@ -1008,7 +1008,7 @@ function alert_details($details)
                 if (in_array($k, ['device_id', 'sysObjectID', 'sysDescr', 'location_id'])) {
                     continue;
                 }
-                if (!empty($v) && (stristr($k, 'id') || stristr($k, 'desc') || stristr($k, 'msg')) && substr_count($k, '_') <= 1) {
```

Comment:
Code, looks ok to me, but could be simplified with str_i_contains()

```
str_i_contains(['id', 'desc', 'msg', 'last'], $k)
```

I'm not sure insensitive is needed either... so maybe could even use Str::contains()

Reply:
changed to it :-)